server/src/schemas/trainingSchema.ts

```diff
@@ -1,5 +1,30 @@
 import { gql } from "apollo-server-express";
 
+export interface TrainingModule {
+  id: number;
+  name: string;
+}
+
+export enum QuestionType {
+  MULTIPLE_CHOICE,
+  CHECKBOXES
```

Comment:
I suggest we rename `QuestionType` to `ModuleItemType` or similar [(see frontend version)](https://github.com/rit-construct-makerspace/makerspace/blob/master/client/src/types/Module.ts#L1-L7). This is because they aren't necessarily questions. They can be information text blocks, YouTube embeds, or images.

Reply:
Yeah I saw a comment about this, will update